front/src/interfaces/organisms/CommunityMenu/ICommunityMenu.ts

```diff
@@ -0,0 +1,3 @@
+export default interface CommunityMenuProps {
+  onMenuChanged: (menu: string) => void;
```

Comment:
```suggestion
  onMenuChanged: (menuItem: string) => void;
```
위의 동작을 보면 menuItem이 더 좋을 것 같습니다!